002-es6-modules.md

```diff
@@ -619,8 +625,8 @@ console.log(namespace.foo); // undefined
 
 // mutate to show 'yo' still exists as a binding
 import cjs_exports from './bad-cjs.js';
-cjs_exports.yo = 'lo again';
-console.log(namespace.yo); // 'yolo again'
+cjs_exports.yo += ' again';
+console.log(namespace.yo); // 'lo again'
```

Comment:
Do you mean you think that would make a better example, or you think that's what the output should be? In both the original and updated code the output would be `lo again`, right? Personally I would probably just make the example `original` / `updated` or something like that.